src/deploy/RPM_build/noobaa.spec

```diff
@@ -51,13 +52,23 @@ ln -s /usr/local/noobaa-core/node/bin/npx $RPM_BUILD_ROOT/usr/local/noobaa-core/
 mkdir -p $RPM_BUILD_ROOT/etc/systemd/system/
 ln %{_builddir}/%{name}-%{version}-%{revision}/noobaa/src/deploy/nsfs.service $RPM_BUILD_ROOT/etc/systemd/system/nsfs.service
 
+cp -R /etc/rsyslog.d $RPM_BUILD_ROOT/etc/rsyslog.d
+cp -R /etc/logrotate.d/noobaa $RPM_BUILD_ROOT/etc/logrotate.d/
+
 
 %files
 /usr/local/noobaa-core
 /etc/systemd/system/nsfs.service
+/etc/logrotate.d/noobaa/logrotate_noobaa.conf
+/etc/rsyslog.d
```

Comment:
I am not sure if I understand this correctly but does this mean that we take the ownership of the entire `/etc/rsyslog.d` directory? What if it is already placed by another program?

Reply:
@tangledbytes ya, It make sense, I will give specific file names,